fastinfoset/src/main/java/com/sun/xml/fastinfoset/Encoder.java

```diff
@@ -735,15 +774,15 @@ protected final void encodeCharacters(char[] ch, int offset, int length) throws
     }
 
     /**
-     * Encode a chunk of Character Information Items.
+     * Encode a chunk of Character Information Items.If the array of characters is to be indexed (as determined by
+    {@link #isCharacterContentChunkLengthMatchesLimit(int)} then the array is not cloned
+ when adding the array to the vocabulary.
      *
-     * If the array of characters is to be indexed (as determined by
-     * {@link #isCharacterContentChunkLengthMatchesLimit(int) } then the array is not cloned
-     * when adding the array to the vocabulary.
      *
      * @param ch the array of characters.
      * @param offset the offset into the array of characters.
      * @param length the length of characters.
+     * @throws java.io.IOException
```

Comment:
I'd add "for errors"; the way it is done just turns one warning into another

Reply:
OK, fixed.